src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/PublicationSummary.test.tsx

```diff
@@ -0,0 +1,332 @@
+      slug: 'rel-1-slug',
+      title: 'Release 1',
+      publicationId: testPublication.id,
+      permissions: {
+        canUpdateRelease: false,
+      },
+      approvalStatus: 'Draft',
+    } as MyRelease,
+    {
+      id: 'rel-3',
+      latestRelease: false,
+      published: '2021-01-01T11:21:17.7585345',
+      slug: 'rel-3-slug',
+      title: 'Amendment Release',
+      amendment: true,
+      previousVersionId: 'rel-2',
+      publicationId: testPublication.id,
+      permissions: {
+        canUpdateRelease: false,
+        canDeleteRelease: true,
+      },
+      approvalStatus: 'Draft',
+    } as MyRelease,
+    {
+      id: 'rel-2',
+      latestRelease: true,
+      published: '2021-05-30T11:21:17.7585345',
+      slug: 'rel-2-slug',
+      title: 'Release 2',
+      publicationId: testPublication.id,
+      permissions: {
+        canUpdateRelease: false,
+      },
+      approvalStatus: 'Approved',
+    } as MyRelease,
+  ];
+
+  const testMethodologies: MyPublicationMethodology[] = [
+    {
+      owner: true,
+      methodology: {
+        amendment: false,
+        id: '1234',
+        latestInternalReleaseNote: 'this is the release note',
+        previousVersionId: 'lfkjdlfj',
+        published: '2021-06-08T09:04:17.9805585',
+        slug: 'meth-1',
+        status: 'Approved',
+        title: 'I am a methodology',
+        owningPublication: {
+          id: 'p1',
+          title: 'Publication title',
+        },
+        permissions: {
+          canApproveMethodology: false,
+          canUpdateMethodology: false,
+          canDeleteMethodology: false,
+          canMakeAmendmentOfMethodology: false,
+          canMarkMethodologyAsDraft: false,
+        },
+      },
+      permissions: {
+        canDropMethodology: false,
+      },
+    },
+  ];
+
+  const testExternalMethodology: MyPublication['externalMethodology'] = {
+    title: 'External methodology',
+    url: 'https://example.com',
+  };
+
+  test('renders correctly with a publication with no releases, methodologies, contact details or permissions', async () => {
+    render(
+      <MemoryRouter>
+        <PublicationSummary
+          publication={testPublication}
+          topicId={testTopicId}
+          onChangePublication={noop}
+        />
+      </MemoryRouter>,
+    );
+
+    expect(screen.getByText('No team name')).toBeInTheDocument();
+    expect(screen.getByText('No contact name')).toBeInTheDocument();
+
+    expect(screen.getByText('No releases created')).toBeInTheDocument();
+
+    expect(screen.getByText('Methodologies')).toBeInTheDocument();
+
+    expect(
+      screen.queryByRole('button', {
+        name: 'Create methodology',
+      }),
+    ).not.toBeInTheDocument();
+
+    expect(
+      screen.queryByRole('button', {
+        name: 'Link to an externally hosted methodology',
+      }),
+    ).not.toBeInTheDocument();
+
+    expect(
+      screen.queryByText('Manage this publication', {
+        selector: 'a',
+      }),
```

Comment:
Similarly to previous cases, we could use the `getByRole/queryByRole('link', { name: 'Manage this publication' })` selector in all of this suite's tests instead of `selector: 'a'`.

Reply:
Done ta